debian/rules

```diff
@@ -46,7 +46,7 @@ override_dh_auto_configure:
 	mkdir -p '$(OUR_GOPATH)/src/github.com/docker'
 	ln -sfT '$(CURDIR)/engine' '$(OUR_GOPATH)/src/github.com/docker/docker'
 
-override_dh_auto_build: _build-tini _build-proxy _build-cli _build-manpages _build-engine
+override_dh_auto_build: _build-tini _build-proxy _build-cli _build-engine
```

Comment:
So the manpages are now part of the `debian/` directory and are being maintained in git, right?

What do you think about generating a stamp file containing the upstream version number referring to the manpages and placing this file inside the `debian/manpages/` directory?  That way, we could implement some sort of check (here on `d/rules`) to make sure that the developer didn't forget to regenerate the docs.  This is not a very elegant thing to do, but I'm afraid that people will forget to run the `build-manpages.sh` script manually and will end up with wrong documentation in the package.